app/utils/pages.tsx

```diff
@@ -13,61 +10,76 @@ interface IPage {
     path: string
 }
 
-export function getPathsToMap (){
-    const pathToLeagueData = path.join(process.cwd(), "app", "leagueConfiguration");
-    const pathsToMap = fs.readdirSync(pathToLeagueData);
-    return pathsToMap;
+interface PageInformation {
+    showStatus: string, 
+    showNameAndSeason: string,
+    friendlyName: string,
+    contestantDataKey: string
 }
 
-export function getLeagueConfigurationData (filename:string){
-    return require(`../leagueConfiguration/${filename}`);
+function constructPageInformation(leagueConfigurationKey:string){
```

Comment:
nit: it might be nice to have an example of what we expect `leagueConfigurationKey` to look like here.

Reply:
This is a great idea. Pushed this up!